### PR TITLE
🤓 simple grammar fix

### PR DESCRIPTION
**Pull request by sebastiansterk**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
It's just a simple grammar fix in some comments & return values. Nothing special.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by sebastiansterk** [comment]
signed.

File: pkg/apis/coordination/validation/validation.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func ValidateLeaseSpec(spec *coordination.LeaseSpec, fldPath *field.Path) field.
 	}
 	if spec.LeaseTransitions != nil && *spec.LeaseTransitions < 0 {
 		fld := fldPath.Child("leaseTransitions")
-		allErrs = append(allErrs, field.Invalid(fld, spec.LeaseTransitions, "must to greater or equal than 0"))
+		allErrs = append(allErrs, field.Invalid(fld, spec.LeaseTransitions, "must be greater or equal than 0"))
```
Comment (KnVerey) [inline pkg/apis/coordination/validation/validation.go:49]:
```suggestion
		allErrs = append(allErrs, field.Invalid(fld, spec.LeaseTransitions, "must be greater than or equal to 0"))
```

**Comment by dims** [comment]
/release-note-none
/ok-to-test
/approve
/lgtm

NOTE: changes in comments or in the error messages, no change in logic.

**Comment by sebastiansterk** [comment]
@KnVerey @dims those tests seems to be flaky + the code has not been changed. Any chance to merge that pr or fix the tests?

**Comment by dims** [comment]
/assign @liggitt 

NOTES: easy/quick to review, unfortunately under pkg/apis so needs api reviewers!

**Pull request merged**